### DISCUSSION TITLE
候选词优先级问题

### DISCUSSION THREAD

**Discussion by ShenJiahuan** [rime/home#1045] [General]
在Mac的鼠须管下（其他平台未测试），三字候选词的优先级高于所有双字，双字高于所有单字，即冷门双字出现在热门单字之前。
请问是否有办法调整这一行为，让一些很冷门的双字优先级降低？谢谢！

**Comment by lotem** [comment]
不按字数排序。
如果按拼音论，排序按照候选与输入码相匹配的音节数多寡排列。首先，拼音与输入码所有音节完全匹配的候选词会排在只匹配部分音节的候选词前面。这是显而易见的。其次就是以（拼音音节序列）越长匹配度越高。

若要按所述要求调整排序，可手动编辑码表，改造成码表式输入方案。相同编码下的字、词会按照字、词频高低有序排列。
